client/components/jetpack-header/liquidweb.jsx

```diff
@@ -0,0 +1,160 @@
+/** @format */
+
+/**
+ * External Dependencies
+ */
+import React from 'react';
+
+function JetpackLiquidWebLogo() {
```

Comment:
Noting that the logo doesn't appear to be horizontally centered properly - it has 17px on the left, but 28 on the right:

![](https://cldup.com/8Mu4e4NhVR.jpg)

Reply:
Good catch, but I also don't know that it has to be perfectly aligned because the shapes on the sides create different kinds of white space. So, in fact, we could invert the spacing and have it optically aligned.